tools/wptrunner/wptrunner/formatters/chromium.py

```diff
@@ -24,12 +24,32 @@ def __init__(self):
         # the trie and the leaf contains the dict of per-test data.
         self.tests = {}
 
-    def _store_test_result(self, name, actual, expected):
+        # Message dictionary, keyed by test name. Value is a concatenation of
+        # the subtest messages for this test.
+        self.messages = defaultdict(str)
```

Comment:
Shall we do this in `test_start` (and we won't need `self.messages.pop`)?

One question is whether it's possible to have multiple tests started in parallel.

Reply:
Yes parallel tests are possible (and that's how we're running on chromium cq), which is why I need to keep a dict of the "active" tests.